client/src/reducers/questionReducer.js

```diff
@@ -1,12 +1,39 @@
-import { SET_QUESTIONS, ANSWER_QUESTION } from '../actions/teamActions';
-import { routerActions } from 'connected-react-router';
+import { SET_QUESTIONS, ANSWER_QUESTION, SET_RESULTS } from '../actions/teamActions';
```

Comment:
should we be using relative paths?

Reply:
yes, I think the ide changed it by accident